desktop/extensions-sdk/build/set-up/minimal-backend-extension.md

```diff
@@ -111,12 +115,9 @@ For more information on the `metadata.json`, see [Metadata](../../extensions/MET
 
 > **Warning**
 >
->Do not replace the `${DESKTOP_PLUGIN_IMAGE}` placeholder in the `metadata.json` file. The placeholder is replaced automatically with the correct image name when the extension is installed.
-{: .warning}
-
+> Do not replace the `${DESKTOP_PLUGIN_IMAGE}` placeholder in the `metadata.json` file. The placeholder is replaced automatically with the correct image name when the extension is installed.
+> {: .warning}
```

Comment:
The warning class is not picked up correctly as the msg is blue instead of orange.

![image](https://user-images.githubusercontent.com/15997951/191443390-1801f2fe-d48e-45e4-9084-c167731fa935.png)

Reply:
Thx foc picking this up. Auto-formatting md files in VS Code is sometimes problematic...

Reply:
This is fixed with latest commit